nautobot_firewall_models/models/capirca_models.py

```diff
@@ -0,0 +1,66 @@
+"""Models for the Capirca Configurations."""
+# pylint: disable=duplicate-code
+import logging
+from django.db import models
+from django.urls import reverse
+from django.utils.module_loading import import_string
+
+from nautobot.extras.utils import extras_features
+from nautobot.core.models.generics import PrimaryModel
+
+from nautobot_firewall_models.utils.capirca import DevicePolicyToCapirca
+from nautobot_firewall_models.constants import PLUGIN_CFG
+
+LOGGER = logging.getLogger(__name__)
+
+
+@extras_features(
+    "custom_fields",
+    "custom_links",
+    "custom_validators",
+    "export_templates",
+    "graphql",
+    "relationships",
+    "statuses",
+    "webhooks",
+)
+class CapircaPolicy(PrimaryModel):
+    """CapircaPolicy model."""
+
+    device = models.OneToOneField(to="dcim.Device", blank=True, null=True, on_delete=models.CASCADE)
+    pol = models.TextField(blank=True, null=True)
+    net = models.TextField(blank=True, null=True)
+    svc = models.TextField(blank=True, null=True)
+    cfg = models.TextField(blank=True, null=True)
+
+    csv_headers = ["device"]
+
+    class Meta:
+        """Meta class."""
+
+        ordering = ["device"]
+        verbose_name_plural = "Capirca Policies"
+
+    def get_absolute_url(self):
+        """Return detail view URL."""
+        return reverse("plugins:nautobot_firewall_models:capircapolicy", args=[self.pk])
+
+    def __str__(self):
+        """Stringify instance."""
+        return self.device.name
```

Comment:
Should there be something that notates it's policy? Not sure how I feel about the repr being a nested objects name.

Reply:
I see what you are saying, but the `pol` is just a few hundred lines of text. Not sure what it should be.

Reply:
Thoughts on something like 

`return f"Capirca Policy -> {self.device.name}"`

Reply:
Perhaps not a big deal, but thought I would mention